### DISCUSSION TITLE
Elastic Fleet in an inconsistent state

### DISCUSSION THREAD

**Discussion by WPTechnician** [Security-Onion-Solutions/securityonion#12172] [2.4]
### Version

2.4.30

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

32

### RAM

144GB

### Storage for /

200GB

### Storage for /nsm

5TB

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

I recently reinstalled SO to move to 2.4 as we had been happily running 2.3 for a few years.  I used the ISO and everything seemed to install cleanly.  I installed the manager/search node first and then a sensor node.  I am seeing alerts from the playbooks fine but I was getting nothing from the Sensor (Metadata or Suricata).  I followed the troubleshooting guide and wasn't able to resolve the issue.  I tried restarting the service on the manager and it says it isn't started.  When I try to start it, it says that it is already started:

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/19175877/ee7d4322-1ca9-4c39-b1fe-a51adeb86d96)

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/19175877/c54e0e7d-bd8d-4321-90b5-5cc533f2bb71)

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/19175877/b5158048-9ac4-4d3f-af13-6de88bd0510a)

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/19175877/fe3cb088-6ff2-49ff-bc55-366661295a53)

The log files in /opt/so/log/elasticfleet are empty.  I am not sure what to do next to get things working.  Thanks!

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by reyesj2** [comment]
On your manager what does  ```salt-call state.apply elasticfleet``` output?

**Reply by WPTechnician** [reply]
Ugh, I think I made things worse for myself.  I uninstalled the Agent on the manager to try and reinstall and it seems it is all just broken.  I am going to reinstall everything and go from there.  Thanks
